src/components/Species/index.tsx

```diff
@@ -388,26 +398,13 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
     }
 
     return params;
-  }, [record, debouncedSearchTerm, selectedOrganization]);
+  }, [record, debouncedSearchTerm, selectedOrganization, searchSortOrder]);
 
   const onApplyFilters = useCallback(
     async (reviewErrors?: boolean) => {
-      const getSpeciesListWithCS = () => {
-        const speciesListWithCS: SpeciesSearchResultRow[] = [];
-        species.forEach((sp) => {
-          speciesListWithCS.push({
-            ...sp,
-            conservationStatus: getConservationStatusString(sp),
-            growthForm: getGrowthFormString(sp),
-            seedStorageBehavior: getSeedStorageBehaviorString(sp),
-            ecosystemTypes: getEcosystemTypesString(sp),
-          });
-        });
-        return speciesListWithCS;
-      };
-
       const params: SearchNodePayload = getParams();
-      if (params.search.children.length > 1) {
+
+      if (species) {
```

Comment:
Added this condition to allow refreshing the search when a species entry is edited